src/sql/workbench/parts/notebook/cellViews/output.component.ts

```diff
@@ -45,9 +45,8 @@ export class OutputComponent extends AngularDisposable implements OnInit {
 	ngOnInit() {
 		this.renderOutput();
 		this._initialized = true;
-		this.cellModel.notebookModel.layoutChanged(() => {
-			this.renderOutput();
-		});
+		this._register(Event.debounce(this.cellModel.notebookModel.layoutChanged, (l, e) => e, 250, /*leading=*/false)
```

Comment:
How did you come up with 250? My only concern is this might be a bit too slow - if you try 50 or 100 do we get any better perf numbers? Would be nice to hit the goldilocks zone of max responsiveness.

The only other thing is that long-term, we probably want to think hard about whether re-rendering the whole is a good thing. Do we have issue tracking need to avoid this and instead just forward resize events through?

Reply:
#5466 to track the longer-term task.

I had used 250 as we have used that elsewhere in the product to limit layout changed calls, but I re-ran some numbers, and using 50 makes ADS more responsive, and still is at a threshold where we're not calling layout() 10+ times/second.